src/images/download.py

```diff
@@ -36,6 +36,8 @@
         'Ubuntu/8.10 (intrepid) Firefox/3.0.5')
 }
 
+logging.getLogger("PIL").setLevel(logging.INFO)
```

Comment:
Agregá un comentario acá de por qué estás tocando algo de PIL, onda "# turning off the PIL debug logs as are too noisy"

Reply:
Got it!